README.md

```diff
@@ -60,20 +71,22 @@ echidna_convert: failed!💥
     mint(57896044618658097711785492504343953926634992332820282019728792003956564819968)
 ```
 
+Echidna also improved the testing of complex contracts with two cool features. First, it [allows initializing a fuzzing campaign with arbitrary transactions using Etheno and Truffle](https://github.com/crytic/echidna/wiki/Deployment-of-a-contract-using-Truffle,-Ganache-and-Etheno-to-test-with-Echidna). Second, it can test more than one contract at the same time, calling any public or external function of any tested contract. Use `multi-abi: true` in your configuration file to allow Echidna to test more than one contract at the same time.
```

Comment:
it doesn't test more than one contract, it just calls any contract with a known abi

Reply:
I think I clarified the sentence. I'm trying to avoid too many technical details in this simplified description of the Echidna features.